docs/xroad_security_server_toolkit_user_guide.md

```diff
@@ -96,9 +97,24 @@ The document is intended for readers with a good knowledge of Linux server manag
 The X-Road Security Server Toolkit package can be installed using PIP:
 
 ```
-$ pip install xrdsst --extra-index-url https://xroad-toolkit.s3-eu-west-1.amazonaws.com/xrdsst
+$ pip install --index-url http://xroad-toolkit.s3-website-eu-west-1.amazonaws.com/xrdsst xrdsst
 ```
 
+The packaged have been signed with a private GPG key:
+```
+$ gpg --detach-sign -a xrdsst-*.tar.gz
+$ gpg --detach-sign -a xrdsst-*.whl
+```
+
+The downloaded and signed packages can be verified with a public key:
+```
+$ gpg --import key.gpg
```

Comment:
I do not see instructions about where to acquire the ``key.gpg``, please add them.

Reply:
Updated